api/endpoints/cmr.py

```diff
@@ -112,14 +116,51 @@ def get(self):
         return respond(resp)
 
 
+@ns.route('/granules/<string:file_uri>/data')
+class CmrGranuleData(Resource):
+    """
+    CMR granule data
+
+        Download granule by file URI
+        file_uri: a UTF-8 encoded URI
+
+        Example:
+        https://api.maap-project.org/api/cmr/granules/https%3A%2F%2Fdata.ornldaac.earthdata.nasa.gov%2Fprotected%2Fgedi%2FGEDI_L3_Land_Surface_Metrics%2Fdata%2FGEDI03_elev_lowestmode_stddev_2019108_2020106_001_08.tif/data
+    """
+
+    def get(self, file_uri):
+        s = requests.Session()
+        response = s.get(parse.unquote(file_uri), stream=True)
+
+        if response.status_code == 401:
+            maap_user = get_authorized_user()
+
+            if maap_user is None:
+                return Response(response.text, status=401)
+            else:
+                urs_token = db.session.query(Member).filter(Member.id == maap_user.id).urs_token
+                s.headers.update({'Authorization': f'Bearer {urs_token},Basic {os.environ.get("MAAP_APP_CREDS")}',
```

Comment:
Where do the `MAAP_APP_CREDS` get set? I don't see them in settings.py or otherwise.